src/location-registry/controllers/helpers.py

```diff
@@ -0,0 +1,26 @@
+from flask import Blueprint, request, jsonify
+import logging
+import datetime as dt
+from bson import json_util, ObjectId
+import json
+from routes import api
+import os
+from http import HTTPStatus
+
+
+_logger = logging.getLogger(__name__)
+monitor_bp = Blueprint('monitor_bp', __name__)
+
+
+@monitor_bp.route(api.route['root'], methods=['GET', 'POST'])
```

Comment:
Since these are more like routes, could we move these to the routes/api or something close to where we are defining our routes?